client/js/libs/handlebars/parse.js

```diff
@@ -56,8 +56,8 @@ function uri(text) {
  * https://github.com/megawac/irc-style-parser/tree/shout
  */
 var styleCheck_Re = /[\x00-\x1F]/,
-    back_re = /^(\d{1,2})(,(\d{1,2}))?/,
-    colourKey = "\x03", colour_re = /\x03/g,
+    back_re = /^([0-9]{1,2})(,([0-9]{1,2}))?/,
```

Comment:
Waouh, I'd pretend I didn't know that, but I also see I +1ed this answer more than 2 years ago :-)